README.md

```diff
@@ -148,7 +148,8 @@ let server = new TChannel({ serviceName: 'server' });
 server.listen(4040, '127.0.0.1');
 let serverThriftChannel = TChannelAsThrift({
   channel: server,
-  entryPoint: path.join(__dirname, 'thrift', 'echo.thrift'), // file path to a thrift file
+  entryPoint: 'thrift/echo.thrift', //  path to a thrift IDL
```

Comment:
I suppose not - I was just trying to make the usage here align with the examples; but yeah it might just be more confusing without context. Will revert.